edgexfoundry/edgex-compose
# fix: add external non secure mqtt broker for taf testing

Closes #287
Signed-off-by: intel <joshua.silverio@intel.com>







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-compose/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>


## Testing Instructions

- Make build-taf
- replace CONF_DIR_PLACE_HOLDER -> /custom-config
- replace MQTT_BROKER_ADDRESS_PLACE_HOLDER -> tcp://edgex-mqtt-taf-broker:1884
- docker-compose -f ../taf/composefile.yml up -d
- try to connect to secure broker edgex-mqtt-broker:1883 (should be not authorized)
- try to connect to edgex-mqtt-taf-broker:1884 (should be allowed as non secure mode)
